interceptors.go

```diff
@@ -44,30 +50,64 @@ func FilterMethodsFunc(ctx context.Context, fullMethodName string) bool {
 	return true
 }
 
-//SetFilterFunc sets the default filter function to be used by interceptors
+// SetFilterFunc sets the default filter function to be used by interceptors
 func SetFilterFunc(ctx context.Context, ff FilterFunc) {
 	if ff != nil {
 		defaultFilterFunc = ff
 	}
 }
 
+// AddUnaryServerInterceptor adds a server interceptor to default server interceptors
+func AddUnaryServerInterceptor(ctx context.Context, i grpc.UnaryServerInterceptor) {
+	unaryServerInterceptors = append(unaryServerInterceptors, i)
+}
+
+// AddStreamServerInterceptor adds a server interceptor to default server interceptors
+func AddStreamServerInterceptor(ctx context.Context, i grpc.StreamServerInterceptor) {
+	streamServerInterceptors = append(streamServerInterceptors, i)
+}
+
+// ReplaceColdBrewServerInterceptors allows enabling/disabling coldbrew server interceptors
+//
+// when set to true, the coldbrew server interceptors will not used
+func ReplaceColdBrewServerInterceptors(ctx context.Context, flag bool) {
+	replaceServerInterceptors = flag
+}
+
+// AddUnaryClientInterceptor adds a server interceptor to default server interceptors
+func AddUnaryClientInterceptor(ctx context.Context, i grpc.UnaryClientInterceptor) {
+	unaryClientInterceptors = append(unaryClientInterceptors, i)
+}
+
+// AddStreamClientInterceptor adds a server interceptor to default server interceptors
+func AddStreamClientInterceptor(ctx context.Context, i grpc.StreamClientInterceptor) {
+	streamClientInterceptors = append(streamClientInterceptors, i)
+}
+
+// ReplaceColdBrewClientInterceptors allows enabling/disabling coldbrew client interceptors
+//
+// when set to true, the coldbrew client interceptors will not used
+func ReplaceColdBrewClientInterceptors(ctx context.Context, flag bool) {
```

Comment:
Can we rename this to UseColdBrewClientInterceptors or UseDefaultClientInterceptors instead for readability?